docs/pages/includes/config-reference/auth-service.yaml

```diff
@@ -156,11 +156,18 @@ auth_service:
         # Possible values: true, false, "hardware_key", "hardware_key_touch".
         # Defaults to false.
         require_session_mfa: false
+        
+        # second_factors is the list of allowed second factors for the cluster.
+        # Possible values: "otp" and "webauthn".
+        # Defaults to ["otp"].
+        second_factors: ["otp"]
```

Comment:
Is this in order of preference?

Is `second_factors: [ otp, webuathn ]` valid?